deploy/awstools/awstools.py

```diff
@@ -22,10 +22,16 @@
 from mypy_boto3_ec2.type_defs import FilterTypeDef
 from mypy_boto3_s3.literals import BucketLocationConstraintType
 
-# setup basic config for logging
+
 if __name__ == '__main__':
+    # setup basic config for logging
     logging.basicConfig()
 
+    # use __builtin.input because we aren't in a StreamLogger context
+    from __builtin__ import input as firesim_input
+else:
+    from util.io import firesim_input
```

Comment:
The import of `firesim_input` is mildly wonky in `awstools.py` because when we use it as a script, we won't be inside a `StreamLogger` and we don't have the path setup correctly to import other firesim python modules.

As an alternative to this janky mess a slightly different flavor of jank could be:
```suggestion
    # add `firesim/deploy` to path so that other firesim modules can be imported
    sys.path.append(__file__ + '../../')

from util.io import firesim_input
```

Other, possibly less janky but more work-filled approaches could be to:
1. create a `firesim/scripts/awstools.py` script that does the sys.path manipulation and remove the `if __name__ == '__main__':` behavior (or change it to exec'ing the wrapper script that knows how to import things).
2. package firesim python library and define `awstools.main()` as an entrypoint.  Defining firesim as a python package would take some work and some rearranging of things.  It would make it installable and we could define entrypoints to the library (CLI scripts) in a way that others familiar with python packages would easily understand.  I'm not suggesting that we do it for this PR, it is only starting to feel like the imports are getting complex enough that it is in the back of my brain as an option.

Reply:
NOTE: I have a vague feeling that the `sys.path.append` might have to be before the first import for it to work.  However, I don't feel like going to find the docs for python importing chicanery this morning.

Reply:
Looks like CI is failing because ```.github/scripts/launch-manager-instance.py``` is trying to import ```awstools```, which fails to import from ```utils```. We'd probably need to also append the ```deploy``` directory to ```PYTHONPATH``` in that script before trying to import ```awstools```, so I think adding ```sys.path.append(ci_workdir + "/deploy")``` to ```.github/scripts/launch-manager-instance.py``` would fix the CI.

Reply:
I like your "slightly different flavor of jank" solution a bit better because it likely won't require other scripts to keep adding ```deploy``` to ```PYTHONPATH```. I'm not sure that the above syntax would work though, I think you'd need to do:

```sys.path.append(os.path.abspath( __file__ ) + '../../')```

Reply:
> it likely won't require other scripts to keep adding deploy to PYTHONPATH

I don't think you're correct here.  Looking at the other scripts in `.github/scripts` in CI that import firesim modules, they would still need to modify their PYTHONPATH, they just need to do it consistently.  The alternative jank implies that an imported module would change the import behavior of `__main__` and I don't think it works that way.